datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/handler/DatalakeFullBackupWaitHandler.java

```diff
@@ -33,7 +33,7 @@
     @Value("${sdx.stack.backup.status.sleeptime_sec:20}")
     private int sleepTimeInSec;
 
-    @Value("${sdx.stack.backup.status.duration_min:90}")
+    @Value("${sdx.stack.backup.status.duration_min:600}")
```

Comment:
10 hours? This looks kind of extreme. This means that you can't to anything else with the DL for that long.
Is this desirable? How long is the backup process in avarage?

Reply:
We have seen cases where the backup/restore was taking 4-5 hours. There were cases where the Solr/Hbase metadata is huge and is taking longer.  We are trying to see ways to optimize Hbase and Solr. Until they land, it will take longer based on the data the customer has.

Reply:
his change is to have the timeout in line with what we have in the datalake-dr service. datalake service is polling for the status from the datalake-dr service.
datalake-dr service now does not timeout till 10 hours to accommodate cases where it is way longer.

Reply:
I understand that the datalake-dr service can live with such long timeouts, but as I've stated this will block other actions for the DL. We don't have a cancel mechanism at the moment. In this situation if I would shoot for the mentioned 4-5 hours.
Also if this is so time sensitive to customer data, maybe there should be a warning and dynamic duration based on the amount of data the customer have.

Reply:
set for 4 hours. I do think a dynamic duration/estimation is something that customers want. as well as a overall on focus on estimation around each part of backup, similar to all the work that went into estimation for UDX, however work is not on any timeline anytime soon